tools/podio-merge-files

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+"""podio-merge-files tool to merge any number of podio files into one"""
+
+import argparse
+import sys
+import podio
+import podio.root_io
+from podio import reading
+
+parser = argparse.ArgumentParser(
+    description="Merge any number of podio files into one, can merge TTree and RNTuple files"
+)
+
+parser.add_argument("--output-file", help="name of the output file", required=True)
+parser.add_argument("files", nargs="+", help="which files to merge")
+parser.add_argument(
+    "--metadata",
+    choices=["none", "all", "first"],
+    default="first",
+    help="metadata to include in the output file, default: "
+    "only the one from the first file, other options: all files, none",
+)
+parser.add_argument(
+    "--metadata-category-name",
+    default="metadata",
+    help="name of the metadata category in the output file, default: metadata",
+)
+parser.add_argument(
+    "--metadata-parameter-name",
+    default="MergeInputFiles",
+    help="name of the metadata category in the output file, default: metadata",
+)
```

Comment:
Does this have to be configurable?

Reply:
I don't know, we could just hardcode it having that string hardcoded there felt out of place when I had another look :shrug:

Reply:
As discussed during the meeting just now. We will hardcode this and the `metadata-category-name` instead of making them configurable.

Reply:
Now they are hardcoded